### ISSUE TITLE
Validation on coco dataset?

### ISSUE THREAD

**Issue by rrbarioni** [michalfaber/keras_Realtime_Multi-Person_Pose_Estimation#102]
Hello @michalfaber ,
Do you have any python code available for evaluating coco metrics on the validation set?
I would like to compare the results I've obtained with yours.

By the way, excellent implementation! Congratulations!

Thanks in advance,

**Comment by michalfaber** [comment]
Hi @rrbarioni, I work on the simplified version of the https://github.com/matteorr/coco-analyze. This is a good repo for the estimation analysis.
